src/game_inventory.cpp

```diff
@@ -1388,6 +1388,23 @@ class read_inventory_preset: public pickup_inventory_preset
         const Character &you;
 };
 
+class Eread_inventory_preset : public read_inventory_preset
```

Comment:
Use `ebookread` instead of `Eread` maybe?

Reply:
Yes "ebookread" would be more consistent. I have troubles with naming things.